libs/barista-components/filter-field/src/filter-field-multi-select/filter-field-multi-select.ts

```diff
@@ -64,19 +63,6 @@ export class DtFilterFieldMultiSelectSubmittedEvent<T> {
 export class DtFilterFieldMultiSelect<T>
   implements DtFilterFieldElement<T>, AfterViewInit
 {
-  /**
-   * Whether the first option should be highlighted when the multi-select panel is opened.
-   * Can be configured globally through the `DT_MULTI_SELECT_DEFAULT_OPTIONS` token.
-   */
-  @Input()
```

Comment:
@ffriedl89, how do you feel about the whole auto focus removal?

The issue is that the Multi-select is kind of broken due to the following:
1- Auto focus only works the first time the panel is opened. If the user closes the panel and then opens it again, no option is auto focused. 
2- While Panel is open, pressing Tab DOESN'T close the panel and just tabs through normally while keeping the panel open.
3- Entering a text which contains a space, while having an option in focus, caused the input not to accept space as a valid character.